manifests/05-dedicated-admins.SubjectPermission.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: dedicated-admins
+  namespace: openshift-rbac-permissions
+spec:
+  subjectKind: Group
+  subjectName: dedicated-admins
+  clusterPermissions:
+    - dedicated-admins-cluster
+  permissions:
+    - clusterRoleName: dedicated-admins-project
+      namespacesAllowedRegex: "*"
+      namespacesDeniedRegex: "(^kube-.|^openshift.|^ops-health-monitoring$|^management-infra$|^default$|^logging$|^sre-app-check$)"
+      allowFirst: true
```

Comment:
we need another permission with the clusterrolename `admin` that has the same allow and deny regex as `dedicated-admins-project`